### PR TITLE
Replace @DATE@ in MAN pages

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: release-tools/mktarball.sh [hunk 1]
```diff
@@ -6,11 +6,30 @@ if [ -z $VERSION ]; then
...
 fi
 DESTDIR=tarsnap-autoconf-${VERSION}
+RELEASEDATE=`date "+%B %d, %Y"`
 
 # Copy bits in
 mkdir ${DESTDIR}
 cp COPYING Makefile.am actarsnap.m4 configure.ac tsserver .autom4te.cfg ${DESTDIR}/
 cp -R keygen keymgmt keyregen lib libarchive libcperciva misc pkg recrypt tar ${DESTDIR}/
+# Copy with substitution
+for f in \
```
Comment (cperciva) [inline release-tools/mktarball.sh]: I generally use upper-case variable names in shell scripts.

File: release-tools/mktarball.sh [hunk 2]
```diff
@@ -6,11 +6,30 @@ if [ -z $VERSION ]; then
...
+	keyregen/tarsnap-keyregen.1-man \
+	keyregen/tarsnap-keyregen.1-mdoc \
+	keygen/tarsnap-keygen.1-man \
+	keygen/tarsnap-keygen.1-mdoc \
+	tar/tarsnap.1-man.in \
+	tar/tarsnap.1-mdoc.in \
+	tar/tarsnap.conf.5-man.in \
+	tar/tarsnap.conf.5-mdoc.in \
+	recrypt/tarsnap-recrypt.1-man \
+	recrypt/tarsnap-recrypt.1-mdoc \
```
Comment (cperciva) [inline release-tools/mktarball.sh]: How about `*/*-man */*-mdoc */*-man.in */*-mdoc.in` here instead of listing all the files individually?

**Comment by gperciva** [comment]
updated

File: release-tools/mktarball.sh [hunk 3]
```diff
@@ -6,11 +6,17 @@ if [ -z $VERSION ]; then
 	exit 1
 fi
 DESTDIR=tarsnap-autoconf-${VERSION}
+RELEASEDATE=`date "+%B %d, %Y"`
 
 # Copy bits in
 mkdir ${DESTDIR}
 cp COPYING Makefile.am actarsnap.m4 configure.ac tsserver .autom4te.cfg ${DESTDIR}/
 cp -R keygen keymgmt keyregen lib libarchive libcperciva misc pkg recrypt tar ${DESTDIR}/
+# Copy with substitution
```
Comment (cperciva) [inline release-tools/mktarball.sh]: Blank line before comment.

File: release-tools/mktarball.sh [hunk 4]
```diff
@@ -6,11 +6,17 @@ if [ -z $VERSION ]; then
...
+for MANPAGE in */*-man */*-mdoc */*-man.in */*-mdoc.in
+do
```
Comment (cperciva) [inline release-tools/mktarball.sh]: Huh.  I suppose this is perfectly valid `sh` syntax, but I've *always* seen it written as `for foo in bar baz; do` on a single line.  Might as well be consistent with 99%+ of other shell scripts.

**Comment by gperciva** [comment]
updated

**Pull request merged**